subtests/example/example.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+#
+# FIXME - add a one-line description of what this script is supposed to test
+#
+# FIXME - possibly add a more verbose multi-line description here. Not
+# strictly necessary but possibly helpful to future maintainers. You
+# could, e.g., include a BZ ID and/or a description of the behavior
+# this test script is intended to check.
+#
+
+###############################################################################
+# BEGIN setup
+#
+# You probably don't want to touch anything here, but you may want
+# to skim it to see the available helper functions
+#
+
+# Don't touch. This lets the docker() wrapper log debug statements
+# to original stdout, while also preserving a context-specific stdout
+exec 5>&1
+
+# In case you need a temp dir, e.g. for bind mounts or Dockerfiles
+TMPDIR=$(mktemp -d --tmpdir docker-test-sample.XXXXXXXX)
+if [ -z "$DEBUG" ]; then
+    trap 'cd /;/bin/rm -rf $TMPDIR' 0
+fi
+
+# Docker image on which to test. This will probably be overridden in ADEPT.
+IMG=centos
```

Comment:
Maybe "IMG" could be passed in as an environment-variable or as an optional parameter?

e.g. 
```bash
[ -n "$IMG" ] || IMG="centos"
```

w/n ADEPT is running this, probably passing in parameters is the best way to go.  Unf. parsing arguments w/in bash really sucks. Esp. if there're anything but trivial single-minus, or positional parameters.  Another place where having a standard "library" or include-file would be super handy.